test/fs.test.js

```diff
@@ -31,6 +31,21 @@ describe('FileSystem based storage provider', function() {
       process.nextTick(done);
     });
 
+    it('should work even it is ran from other path', function(done) {
+      process.chdir('../../../');
+
+      try {
+        console.log(`running from ${process.cwd()}`);
+        client = new FileSystemProvider({
+          root: path.join(__dirname, 'storage'),
+        });
+
+        process.nextTick(done);
+      } catch (error) {
```

Comment:
`process.nextTick(done(error))` should work

Reply:
Hello @jannyHou, this means that a should remove the `try catch` or replace `process.nextTick(done.bind(null, 'Error'));` with `process.nextTick(done(error));`?

Reply:
Changed `process.nextTick(done.bind(null, 'Error'));` to `process.nextTick(done(error));`

Reply:
> Changed process.nextTick(done.bind(null, 'Error')); to process.nextTick(done(error));

Thank you, exactly, since you expect 
```ts
client = new FileSystemProvider({
          root: path.join(__dirname, 'storage'),
        });
``` 
to pass, if it fails with any `error`, the error details should be passed into `done()` so that mocha can report it.